julia-mode/ife

```diff
@@ -0,0 +1,10 @@
+# -*- mode: snippet -*-
+# name: if ... ... else ... end
+# key: ife
+# --
+if ${1:cond}
+    ${2:true}
```

Comment:
Please add \`%\` before `${2:true}` to give this snippet implicit selection support.

```diff
-  ${2:true}
+  `%`${2:true}
```